internal/quickstart/container.go

```diff
@@ -69,7 +69,9 @@ func (m ContainerModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 		case key.Matches(msg, pressableKeys.Back):
 			switch m.currentStep {
 			case stepCreateFlag:
-				// can't go back
+				// can only go back if a flag has been created but not confirmed,
+				// so let the model handle the Update
+				m.currentModel, cmd = m.currentModel.Update(msg)
```

Comment:
this is new also, to allow going back after a flag has been created (but not before)